### PR TITLE
Added config option to provide preference list for Connection policy

### PR DESCRIPTION
**Pull request by shettyh**

### CODE REVIEW

File: benchmark/src/main/java/com/microsoft/azure/cosmosdb/benchmark/Configuration.java [hunk 1]
```diff
@@ -242,6 +246,7 @@ ConnectionPolicy getConnectionPolicy() {
         ConnectionPolicy policy = new ConnectionPolicy();
         policy.setConnectionMode(connectionMode);
         policy.setMaxPoolSize(maxConnectionPoolSize);
+        policy.setPreferredLocations(preferredLocations);
```
[the remark this answers is not in the document]
Comment (shettyh) [reply benchmark/src/main/java/com/microsoft/azure/cosmosdb/benchmark/Configuration.java]: Fixed

**Comment by moderakh** [comment]
@kushagraThapar this might require porting to v4.

**Pull request merged**